api/src/wfl/module/covid.clj

```diff
@@ -281,7 +284,8 @@
 (def ^:private terra-workspace-sink-serialized-fields
   {:workspace   :workspace
    :entity      :entity
-   :fromOutputs :from_outputs})
+   :fromOutputs :from_outputs
+   :identifier  :identifier})
 
```

Comment:
We need `-sink-name`, `-sink-type`, and `-sink-table`, and `TerraWorkspaceSinkDetails`?
I'm so confused.

Reply:
The`name` is the user friendly name
The type is the internal developer type name. This may change and may not make sense in the future so the user name abstracts it.
The table is the postgres table that stores the metadata of the sink
The details is the name of an sql type. we instantiate one per workload and it records all the workflows we've sunk from the executor